### ISSUE TITLE
[119] Visual Instruction Tuning

### ISSUE THREAD

**Issue by long8v** [long8v/PTIR#128]
[paper]( )

## TL;DR
- **I read this because.. :**
- **task :**
- **problem :**
- **idea :**
- **input/output :**
- **architecture :**
- **objective :**
- **baseline :**
- **data :** 
- **evaluation :**
- **result :**
- **contribution :**
- **etc. :**

## Details
https://llava.hliu.cc/
demo랑 놀아보았다
<img width="960" alt="image" src="https://github.com/long8v/PTIR/assets/46675408/431de7ba-f0fb-4d4b-a975-256cbb154bda">

<img width="971" alt="image" src="https://github.com/long8v/PTIR/assets/46675408/7d755617-be67-441d-9fea-89889657586f">

<img width="972" alt="image" src="https://github.com/long8v/PTIR/assets/46675408/fc82b08f-4ddc-4494-b74b-e21eb02a9493">

<img width="981" alt="image" src="https://github.com/long8v/PTIR/assets/46675408/ccd16297-657b-4625-b6af-cb93551ffeee">

predicate가 더이상 동사가 아님..
<img width="984" alt="image" src="https://github.com/long8v/PTIR/assets/46675408/7803f163-bbf8-4db4-b1c2-2f5ac454469e">

거짓말 시작..

예시를 잘못 줘도 그냥 답변에 포함. 그래도 대충 말은 되는 triplet을 만드는군
<img width="959" alt="image" src="https://github.com/long8v/PTIR/assets/46675408/1ae11635-62cd-400e-93a6-8b63f007bd17">

<img width="932" alt="image" src="https://github.com/long8v/PTIR/assets/46675408/f06bd7d4-18fa-4177-84d5-e28c7811f10d">

여기도 hallucination이 ..  visual genome은 아마 학습 데이터에 있었을 것 같으니 다른 데이터 가져와보자
내가 대만에서 찍은 이 사진 ..

<img width="490" alt="image" src="https://github.com/long8v/PTIR/assets/46675408/b4bf4bf5-fe6b-4c2c-ad68-81ecf107df2e">

<img width="951" alt="image" src="https://github.com/long8v/PTIR/assets/46675408/7bde0a9b-8376-4a76-b8bb-294f00991332">

child가 어딨는지 모르겠지만 대충 맞다

<img width="975" alt="image" src="https://github.com/long8v/PTIR/assets/46675408/d5a72445-4aeb-4742-8bdb-461d44b7491f">

<img width="669" alt="image" src="https://github.com/long8v/PTIR/assets/46675408/d775d09b-4f96-4d2d-b9ce-dd0cae1f6097">

나름 clear한 sample인데

<img width="980" alt="image" src="https://github.com/long8v/PTIR/assets/46675408/d36c4342-071f-47b2-a6f2-efeee2a4ad88">

점점 맛가기 시작 ㅜㅜ 벤치마크에 오버피팅 된 듯한 

<img width="965" alt="image" src="https://github.com/long8v/PTIR/assets/46675408/b2e1b227-7cfd-4ee1-9d25-a4350eab306d">

prompt를 바꾸니 갑자기 또 바른 말 하기 시작...

예시를 잘 주니 나름 잘 동작 그러나 애기는 어디있는가?
<img width="958" alt="image" src="https://github.com/long8v/PTIR/assets/46675408/26035797-a5dc-49b7-8e3a-db317f950df7">

<img width="954" alt="image" src="https://github.com/long8v/PTIR/assets/46675408/27d3b6cb-90bb-4d0e-8d86-9cebc6cd468c">

완벽하네
<img width="973" alt="image" src="https://github.com/long8v/PTIR/assets/46675408/98e7557c-e455-4c50-8081-7b6ca7aa32f1">

약간 동어 반복이긴 하지만 틀린 말은  안하네

<img width="956" alt="image" src="https://github.com/long8v/PTIR/assets/46675408/43308a72-acf5-4df8-9fdc-b8d0d5f6d650">

감성적인 말까지 ..

<img width="979" alt="image" src="https://github.com/long8v/PTIR/assets/46675408/ea9670aa-4d69-42bf-8ff3-7a039f63b5f4">
<img width="967" alt="image" src="https://github.com/long8v/PTIR/assets/46675408/d966a85b-a618-4fc7-9b7a-f3cf361310da">

3절 4절 뇌절해도 잘받아줌..

여기까지 놀고 논문은 다음에 읽어보도록 하자..

**Comment by long8v** [comment]
vit 10번째 layer feature > 12 layer oversmoothing?!
